src/components/Tabs/FeatureThresholdsTab.tsx

```diff
@@ -127,6 +125,14 @@ export default function FeatureThresholdsTab(inputProps: FeatureThresholdsTabPro
   const { scrollShadowStyle, onScrollHandler, scrollRef } = useScrollShadow();
   const selectContainerRef = useRef<HTMLDivElement>(null);
 
+  const store = useViewerStateStore(
+    useShallow((state) => ({
+      thresholds: state.thresholds,
+      setThresholds: state.setThresholds,
+      dataset: state.dataset,
+    }))
+  );
+
```

Comment:
This and the removal of some props were the only real changes in this file. Everything else is just renaming of variables to use `store` instead of `props`.